addons/core/functions/fnc_getSwSettings.sqf

```diff
@@ -24,24 +24,33 @@ params[["_radio","",[""]]];
 private _value = TFAR_RadioSettingsNamespace getVariable _radio;
 if (!isNil "_value") exitWith {_value};
 
-
-
-if (!(TF_use_saved_sw_setting) or (isNil "TF_saved_active_sw_settings")) then {
+if ((isNil "TF_saved_active_sw_settings") || {!TF_use_saved_sw_setting} then  {
     private _defaultRadios = call TFAR_fnc_getDefaultRadioClasses;
     private _parent = getText (configFile >> "CfgWeapons" >> _radio >> "tf_parent");
     if ((_defaultRadios select 1) == _parent or {(_defaultRadios select 2) == _parent}) then {
```

Comment:
Could maybe move all this into fnc_getDefaultFrequencies. Which takes a bool as parameter to differentiate between SR and LR.